x-pack/plugins/cases/public/components/severity/config.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useEuiTheme, EuiHealth } from '@elastic/eui';
+import { CaseSeverity } from '../../../common/types/domain';
+import { CRITICAL, HIGH, LOW, MEDIUM } from './translations';
+
+interface Props {
+  severity: 'low' | 'medium' | 'high' | 'critical';
```

Comment:
should this be an enum? Ideally we would use the enum defined in x-pack/plugins/cases/common/types/domain/case/v1.ts

Reply:
changed to be CaseSeverity from /types/domain